src/main/scala/org/apache/spark/sql/delta/storage/HDFSLogStore.scala

```diff
@@ -118,6 +121,18 @@ class HDFSLogStore(sparkConf: SparkConf, defaultHadoopConf: Configuration) exten
     new Path(path.getParent, s".${path.getName}.${UUID.randomUUID}.tmp")
   }
 
+  private def mayRemoveCrcFile(fc: FileContext, path: Path): Unit = {
```

Comment:
Nit: Can we also rename this to something like tryRemoveCrcFile or cleanupCrcFile?